pkg/storageops/aws/aws.go

```diff
@@ -146,10 +146,9 @@ func (s *ec2Ops) waitAttachmentStatus(
 	request := &ec2.DescribeVolumesInput{VolumeIds: []*string{&id}}
 	actual := ""
 	interval := 2 * time.Second
-	fmt.Printf("Waiting for state transition to %q", desired)
```

Comment:
Any reason why we want to remove this code ? It helps debugging AWS ebs attach issues.

Reply:
If we need it, then let's add it to the log, not fmt.Printf(). Is that ok?

Reply:
+1 for logrus instead of fmt.